themes/bootstrap3/templates/Auth/Database/recovery.phtml

```diff
@@ -1,18 +1,12 @@
 <div class="form-group">
-  <label class="col-sm-3 control-label"><?=$this->transEsc('recovery_by_username') ?>:</label>
-  <div class="col-sm-9 col-md-5">
-    <input type="text" name="username" class="form-control"/>
-  </div>
+  <label class="control-label"><?=$this->transEsc('recovery_by_username') ?>:</label>
+  <input type="text" name="username" class="form-control"/>
 </div>
 <div class="form-group">
-  <label class="col-sm-3 control-label"><?=$this->transEsc('recovery_by_email') ?>:</label>
-  <div class="col-sm-9 col-md-5">
-    <input type="email" name="email" class="form-control"/>
-  </div>
+  <label class="control-label"><?=$this->transEsc('recovery_by_email') ?>:</label>
+  <input type="email" name="email" class="form-control"/>
 </div>
 <?=$this->recaptcha()->html($this->useRecaptcha) ?>
 <div class="form-group">
-  <div class="col-sm-9 col-sm-offset-3">
-    <input class="btn btn-default" name="submit" type="submit" value="<?=$this->transEsc('Recover Account') ?>"/>
-  </div>
-</div>
\ No newline at end of file
+  <input class="btn btn-default" name="submit" type="submit" value="<?=$this->transEsc('Recover Account') ?>"/>
```

Comment:
Not directly related to this PR, but I notice this is coming through as a plain generic button, which looks inconsistent with the stylized buttons in other parts of the UI. Intentional or oversight?